src/modules/AdvancedPaste/AdvancedPaste/AdvancedPasteXAML/Pages/MainPage.xaml.cs

```diff
@@ -86,6 +87,10 @@ public async void LoadClipboardHistoryAsync()
 
                 _dispatcherQueue.TryEnqueue(async () =>
                 {
+                    clipboardHistory.Where(x => x.Image is not null)
```

Comment:
```suggestion
                    // Clear to avoid leaks due to Garbage Collection not clearing the bitmap from memory. Fix for https://github.com/microsoft/PowerToys/issues/33423
                    clipboardHistory.Where(x => x.Image is not null)
```